templates/cli/lib/commands/init.js.twig

```diff
@@ -40,12 +40,35 @@ const initProject = async () => {
         log('You must login first')
         await loginCommand();
     }
+    let answers = {};
 
-    const answers = await inquirer.prompt(questionsInitProject)
-    if (answers.override === false) {
-        process.exit(1)
+    if (organizationId && projectId && projectName) {
+        answers = {
+            project: { id: projectId, name: projectName },
+            organization: { id: organizationId },
+            start: 'existing'
+        }
+
+        try {
+            await projectsGet({ projectId, parseOutput: false });
+        } catch (e) {
+            if (e.code === 404) {
+                answers.start = 'new';
+                answers.id = answers.project.id;
+                answers.project = answers.project.name;
+            } else {
+                throw e;
+            }
+        }
+
+    } else {
+        answers = await inquirer.prompt(questionsInitProject)
```

Comment:
Hmm. What if we did different approach?
We start with empty "answers" object.. We try to fill ith with params from CMD, only using what is provided. We then take `questionsInitProject` and filter out questions that we already have answer for. Then, we show remaining questions and  fill them into our answers object. If all answers were already given with params, there is no need to ask questions all-together

Reply:
Addressed